src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -1267,10 +1267,6 @@ PeerImp::onMessage (std::shared_ptr <protocol::TMProposeSet> const& m)
     if (set.has_hops() && ! slot_->cluster())
         set.set_hops(set.hops() + 1);
 
-    // VFALCO Magic numbers are bad
-    if ((set.closetime() + 180) < app_.timeKeeper().closeTime().time_since_epoch().count())
-        return;
-
```

Comment:
Is there value to contining to filter old proposals but allowing an exception for "agree to disagree" times? I.e. `if (set.closetime() > 0 && ...`

Reply:
My personal take is that we should not filter high-level things at such a low level.

Not only is it flakey to have such special cases—we wouldn’t be discussing this if it weren’t—but we don’t really know what the right cutoff value is.

Is 3 minutes right? Maybe, maybe not. Higher level code uses a different interval for “stale” proposals, which means that this code will reject incoming proposals higher level code would have accepted and considered. 

This commit can result in a dispatch of some stale proposals that would be discarded without it, but I don’t believe this to be significant.